### PR TITLE
fix spelling mistakes in several modules

### PR DESCRIPTION
**Pull request by rndmh3ro**
##### SUMMARY
fix spelling mistakes in vmware module

##### ISSUE TYPE
 - Bugfix Pull Request
 - Docs Pull Request

##### COMPONENT NAME
vmware_guest

##### ANSIBLE VERSION

```
devel
```

### CODE REVIEW

**Review by mkrizek — APPROVED** [review]

File: lib/ansible/modules/cloud/vmware/vmware_guest.py [hunk 1]
```diff
@@ -909,7 +909,7 @@ def sanitize_network_params(self):
                                           " a VLAN name under VM network list.")
 
             if 'name' in network and find_obj(self.content, [vim.Network], network['name']) is None:
-                self.module.fail_json(msg="Network '%(name)s' does not exists" % network)
+                self.module.fail_json(msg="Network '%(name)s' does not exist." % network)
```
Comment (mkrizek) [inline lib/ansible/modules/cloud/vmware/vmware_guest.py:912]: The integration tests need to be changed as well.

**Review by mkrizek — CHANGES_REQUESTED** [review]

File: lib/ansible/modules/cloud/vmware/vmware_guest.py (hunk 1 above)
Comment (rndmh3ro) [reply lib/ansible/modules/cloud/vmware/vmware_guest.py:912]: Can you tell me where to find these?
Searching for "does not exists" didn't give any results.
However I found more spelling mistakes that I can fix.

Comment (Akasurde) [reply lib/ansible/modules/cloud/vmware/vmware_guest.py]: https://github.com/ansible/ansible/blob/devel/test/integration/targets/vmware_guest/tasks/network_negative_test.yml#L68

Comment (Akasurde) [reply lib/ansible/modules/cloud/vmware/vmware_guest.py:912]: https://github.com/ansible/ansible/blob/f8b85bdbf3b1ee4ac1a80cd6da852d1a1ab1a2fe/test/integration/targets/vmware_guest/tasks/network_negative_test.yml#L68

Comment (rndmh3ro) [reply lib/ansible/modules/cloud/vmware/vmware_guest.py:912]: @mkrizek , done.

**Comment by Akasurde** [comment]
@mkrizek Could you please review this again ?

**Review by gundalow — APPROVED** [review]

**Comment by gundalow** [comment]
Looks good, please cherry-pick into `stable-2.5`

**Review by Akasurde — APPROVED** [review]

**Comment by Akasurde** [comment]
@gundalow OK.

**Review by mwperina — APPROVED** [review]
Looks good to me

**Review by mkrizek — APPROVED** [review]

**Comment by nitzmahone** [comment]
cherry-picked to stable-2.5 for 2.5.0rc3

**Pull request merged**